xclim/indices/_hydrology.py

```diff
@@ -2,11 +2,12 @@
 from __future__ import annotations
 
 import numpy as np
-import xarray
+import xarray as xr
```

Comment:
This is fine. Don't forget to find and replace all instances of `xarray`

Reply:
This issue is why the linting test is failing.